packages/serialport/lib/parsers.js

```diff
@@ -2,7 +2,6 @@ module.exports = {
   ByteLength: require('@serialport/parser-byte-length'),
   CCTalk: require('@serialport/parser-cctalk'),
   Delimiter: require('@serialport/parser-delimiter'),
-  InterByteTimeout: require('@serialport/parser-inter-byte-timeout'),
```

Comment:
if i add the package here, `npm run test` fails. package not found... what should i do to make the package internally visible?